front/src/modules/trainschedule/components/Timetable/TimetableManageTrainSchedule.tsx

```diff
@@ -10,8 +9,10 @@ import {
   SubmitConfAddTrainSchedule,
   SubmitConfUpdateTrainSchedules,
 } from 'modules/trainschedule/components/ManageTrainSchedule';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports in the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done